packages/mosaic/tree/tree-selection.component.ts

```diff
@@ -174,12 +169,19 @@ export class McTreeSelection extends CdkTree<McTreeOption>
         differs: IterableDiffers,
         changeDetectorRef: ChangeDetectorRef,
         @Attribute('tabindex') tabIndex: string,
-        @Attribute('multiple') multiple: string
+        @Attribute('multiple') private multipleAttr: string
```

Comment:
You should delete `private` (we discused it), also you should delete suffix `Attr`, in this scope decorator `@Attribute` said that it attribute.

Reply:
fair enough - fixed